src/core_plugins/timelion/public/directives/timelion_expression_input_helpers.js

```diff
@@ -1,18 +1,35 @@
 import _ from 'lodash';
 
-export class FunctionSuggestions {
+export const SUGGESTION_TYPE = {
+  ARGUMENTS: 'arguments',
+  ARGUMENT_VALUE: 'argument_value',
```

Comment:
Is this value ever used anywhere besides line 103? Do we need it?

Reply:
It lays the ground work for argument value suggestions. This PR does not provide those but I wanted to clearly make where they will go.

Reply:
Got it! Maybe we could add a `TODO` comment here then?